### ISSUE TITLE
Предложение

### ISSUE THREAD

**Issue by XionioXMaster** [thexaero/open-parties-and-claims#103]
Добавить отображение всех игроков на глобальной карте при включённом админ-моде !

**Comment by XionioXMaster** [comment]
> Thanks for the suggestion. Not going to use the admin mode because that is a claim, not a party, feature but I could do something with this.

I also propose to make the right to allocate more chunks to a certain group

**Comment by thexaero** [comment]
@XionioXMaster That should've been a separate issue instead of making a comment here. It's already a feature though. You can do it with FTB Ranks.
```
#The FTB Ranks permission that should override the default "maxPlayerClaims" value. Set it to an empty string to never check permissions.
maxPlayerClaimsFTBPermission = "xaero.pac_max_claims"

#The FTB Ranks permission that should override the default "maxPlayerClaimForceloads" value. Set it to an empty string to never check permissions. The permission override only takes effect after the player logs in at least once after a server (re)launch, so it is recommended to keep all permission-based forceload limits equal to or greater than "maxPlayerClaimForceloads".
maxPlayerClaimForceloadsFTBPermission = "xaero.pac_max_forceloads"
```
Or you can give bonus claims to specific people in the UI.

**Comment by XionioXMaster** [comment]
> @XionioXMaster That should've been a separate issue instead of making a comment here. It's already a feature though. You can do it with FTB Ranks.
> 
> ```
> #The FTB Ranks permission that should override the default "maxPlayerClaims" value. Set it to an empty string to never check permissions.
> maxPlayerClaimsFTBPermission = "xaero.pac_max_claims"
> 
> #The FTB Ranks permission that should override the default "maxPlayerClaimForceloads" value. Set it to an empty string to never check permissions. The permission override only takes effect after the player logs in at least once after a server (re)launch, so it is recommended to keep all permission-based forceload limits equal to or greater than "maxPlayerClaimForceloads".
> maxPlayerClaimForceloadsFTBPermission = "xaero.pac_max_forceloads"
> ```
> 
> Or you can give bonus claims to specific people in the UI.

Using LuckyPerms is much more convenient and now everyone is actively switching to it

**Comment by thexaero** [comment]
Then you don't have this feature, I guess.

**Comment by XionioXMaster** [comment]
> Then you don't have this feature, I guess.



> Then you don't have this feature, I guess.

I also found a nuance in fashion, players can see all the privates of other players on the global map, which is why they can kill them, as for me this is a huge problem

**Comment by thexaero** [comment]
You can change it so that only their own claims and server claims are synced to the players.

**Comment by XionioXMaster** [comment]
> You can change it so that only their own claims and server claims are synced to the players.

Thank you, is it also possible to somehow remove a chunk away from the player? For example, I cannot delete a chunk if it is not nearby, but it is essentially mine

**Comment by XionioXMaster** [comment]
> You can change it so that only their own claims and server claims are synced to the players.

It seems to have understood, but if a player is in a group, he does not see his friend's private
Or, for example, the privates of allies

**Comment by thexaero** [comment]
It is what it is. It's not possible to also include party members and allies. Filtering would require too much resources on big parties with a lot of allies.

**Comment by thexaero** [comment]
Maybe filtering isn't that bad, now that I think about it, but people joining and leaving parties would be a nightmare. Seems very difficult to code it and would use a lot of resources to sync everything when people join or leave parties. Not worth it overall.

**Comment by XionioXMaster** [comment]
> It seems to have understood, but if a player is in a group, he does not see his friend's private
> Or, for example, the privates of allies

Hmm, there are nuances. Let's say some player makes holes near the player's chunks (that is, creates 1 chunk around the main one), because of which the 2nd player will not be able to find out where the other person has privated the chunk, even the admin will not know where the player's private is on the map, which is why confusion results.

I propose to add a display of the boundaries of the chunks that players visit during the game, as soon as the player leaves the chunks, they will disappear from the map.

**Comment by XionioXMaster** [comment]
Also, I did not find permission to ignore privates, maybe there is one, but I did not find it, it will be half useful for admins. Since issuing everyone an OP is DANGEROUS

**Comment by thexaero** [comment]
> Hmm, there are nuances. Let's say some player makes holes near the player's chunks (that is, creates 1 chunk around the main one), because of which the 2nd player will not be able to find out where the other person has privated the chunk, even the admin will not know where the player's private is on the map, which is why confusion results.
> 
> I propose to add a display of the boundaries of the chunks that players visit during the game, as soon as the player leaves the chunks, they will disappear from the map.

I'm not sure if I understand most of what you mean here. Did you mean to put the first paragraph after the second? I'll consider adding a permission for syncing all claims even if it's set to owned-only, so that admins can admin better. 

And I like your idea about displaying the boundaries of the claim that you're in, but it might be an overkill. I think a simpler and more practical way would be to just sync a certain radius around the player. That way the hole problem won't exist.

And a separate permission for admin mode can be added too I guess.

**Comment by XionioXMaster** [comment]
> > Hmm, there are nuances. Let's say some player makes holes near the player's chunks (that is, creates 1 chunk around the main one), because of which the 2nd player will not be able to find out where the other person has privated the chunk, even the admin will not know where the player's private is on the map, which is why confusion results.
> > I propose to add a display of the boundaries of the chunks that players visit during the game, as soon as the player leaves the chunks, they will disappear from the map.
> 
> I'm not sure if I understand most of what you mean here. Did you mean to put the first paragraph after the second? I'll consider adding a permission for syncing all claims even if it's set to owned-only, so that admins can admin better.
> 
> And I like your idea about displaying the boundaries of the claim that you're in, but it might be an overkill. I think a simpler and more practical way would be to just sync a certain radius around the player. That way the hole problem won't exist.
> 
> And a separate permission for admin mode can be added too I guess.

I used a translator, but I'll try to explain it more simply, It would be nice if the private appeared on the player's map when he visited, so that groups or allies could at least orient themselves in the private of their comrades. Or add the ability for players to turn off the visibility of private on the global map, which sounds more interesting

**Comment by thexaero** [comment]
I won't add a toggle for visibility for each player. That makes it too easy to lag the server out. All of their claims will have to be resynced to every other player every time a player toggles visibility.
